policyengine_us/variables/gov/states/ar/tax/income/ar_taxable_capital_gains.py

```diff
@@ -14,12 +14,10 @@ def formula(person, period, parameters):
         lt_capital_gains = person("long_term_capital_gains", period)
         # Line 4-6 - short term capital loss
         st_capital_gains = person("short_term_capital_gains", period)
-        capital_loss = -st_capital_gains
-        has_capital_loss = capital_loss > 0
+        st_capital_loss = max_(-st_capital_gains, 0)
+        has_capital_loss = st_capital_gains < 0
```

Comment:
Let's distinguish st vs lt throughout
```suggestion
        has_st_capital_loss = st_capital_loss > 0
```